SPEC.md

```diff
@@ -2460,6 +2604,8 @@ Would produce the command `python script.py --bams=/path/to/1.bam,/path/to/2.bam
 
 ##### Array serialization using write_lines()
 
+:pig2: [Cromwell supported](https://github.com/broadinstitute/cromwell#wdl-support) :white_check_mark:
```

Comment:
I am an advocate of specifically also pointing out parts that Cromwell does not support. Something like "planned for a future version of Cromwell" to tell the user that this is not currently functional. Alternatively, we could remove any sections that are not currently implemented. If we go with the latter option, these Cromwell supported bits can be removed, since anything in the document should be supported.

The reason why I point this out is if I am linked to a specific part of the spec, and I don't read other parts, a lack of `Cromwell supported` checkbox isn't very noticeable.